examples/graphics/lights-baked-render.html

```diff
@@ -155,12 +157,21 @@
 
         // bake lightmaps
         app.lightmapper.bake(null, bakeType);
+//        app.lightmapper.bake([ground], bakeType);
 
         // Set an update function on the app's update event
         var time = 4;
+        var bake = true;
         app.on("update", function (dt) {
             time += dt;
 
+            if (time > 7 && bake) {
```

Comment:
What this delay is for?

Reply:
it's just my test code while refactoring to re-bake one one time to test memory release and stuff. It won't be there when done.